package.json

```diff
@@ -39,5 +40,9 @@
     "transform": {
       "^.+\\.jsx?$": "babel-jest"
     }
-  }
+  },
+  "pre-commit": [
+    "test",
+    "build:storybook"
+  ]
```

Comment:
This might be better as a pre-push step.  It's easy enough to change though so let's try this for now.